### ISSUE TITLE
[BUG] publish libnpmaccess @ 6.0.2

### ISSUE THREAD

**Issue by theladyjaye** [npm/cli#4587]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

8.5.5 has a requirement of libnpmaccess@^6.0.2 but only 6.0.1 is currently published.

Dependency:
https://github.com/npm/cli/blob/latest/package.json#L82

https://www.npmjs.com/package/libnpmaccess

### Expected Behavior

Publish libnpmaccess 6.0.2

### Steps To Reproduce

Observe package.json for 8.5.5 and npm

https://github.com/npm/cli/blob/latest/package.json#L82

https://www.npmjs.com/package/libnpmaccess

**Comment by kotslee** [comment]
As a workaround, you can specify an exact version of npm on your package.json so it will not use the latest 8.5.5 version.
    "npm": "8.5.4",

**Comment by theladyjaye** [comment]
I could, but I don't control the package that lists "npm" as the dependency:

https://github.com/semantic-release/npm/blob/master/package.json#L28

**Comment by andreladocruz** [comment]
@aventurella , you can set on the package.json

```
  "engines": {
    "node": "14.x",
    "npm": "8.5.4"
  },
```

**Comment by wraithgar** [comment]
Is this actually causing something to fail? npm bundles it dependencies so this shouldn't be making any installs fail. I can currently install semantic-release and npm without errors.

We will get that published soon.

**Comment by jeemok** [comment]
installing `npm install -g npm@8.5.4` nor setting in the `package.json` for `"npm": "8.5.4"` isn't working for me, still seeing this prompt:

<img width="558" alt="Screenshot 2022-03-18 at 11 39 31 AM" src="https://user-images.githubusercontent.com/26216852/158933312-7f1a8107-2673-4665-9c13-6c47c2f3115f.png">

choosing either of the options would fail later in the CI/CD pipeline when using the lock file, as it will cause a reference error:
```
libnpmaccess@^6.0.2:
  version "6.0.1"
  resolved "https://registry.yarnpkg.com/libnpmaccess/-/libnpmaccess-6.0.1.tgz#76a6xxxx"
  integrity sha512-xxx
  dependencies:
    aproba "^2.0.0"
    minipass "^3.1.1"
    npm-package-arg "^9.0.0"
    npm-registry-fetch "^13.0.0"
```

<img width="642" alt="Screenshot 2022-03-18 at 11 43 08 AM" src="https://user-images.githubusercontent.com/26216852/158933641-8ace4b2e-5f8c-4be3-8006-e5d580e54c08.png">

can we prioritize this, please

**Comment by ruyadorno** [comment]
Published `libnpmaccess@6.0.2`, please let us know if that fixes it for you.

**Comment by jeemok** [comment]
it works now, thank you @ruyadorno!
